zfs-prune-snapshots

```diff
@@ -353,15 +382,20 @@ for line in "${lines[@]}"; do
 	ht=$(human-time "$delta")
 	hu=$(human-size "$used")
 
+    echo ""
 	if $dryrun; then
 		echo -n '[dry-run] '
 	fi
 
 	echo "[$i/$numsnapshots] removing $snapshot: $ht old ($hu)"
 
-	if ! $dryrun; then
-		zfs destroy "${destroyargs[@]}" "$snapshot" || code=1
-	fi
+	zfs destroy "${destroyargs[@]}" "$snapshot"
+    code=$?
```

Comment:
wrong indentation here and on line 385?

Reply:
I fixed the identation using tabs instead of spaces